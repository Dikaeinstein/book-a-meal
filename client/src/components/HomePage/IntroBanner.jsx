import React from 'react';
import Button from '../util/Button';

const IntroBanner = () => (
  <section className="text-center intro-banner image-overlay text-white">
    <h1>Book-A-Meal</h1>
    <p style={{ margin: '1rem' }}>Your favourite meal order service</p>
    <a href="#menu">
      <Button
        value="Book A Meal Now"
        className="btn btn-intro-lg"
      />
    </a>
  </section>
);

export default IntroBanner;
