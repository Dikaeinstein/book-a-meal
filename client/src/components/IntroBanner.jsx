import React from 'react';
import Button from './Button';

const IntroBanner = () => (
  <section className="text-center intro-banner image-overlay">
    <h1>Book-A-Meal</h1>
    <p>Your favourite meal order service</p>
    <a href="#menu">
      <Button
        value="Book A Meal Now"
        className="btn btn-intro-lg"
      />
    </a>
  </section>
);

export default IntroBanner;
