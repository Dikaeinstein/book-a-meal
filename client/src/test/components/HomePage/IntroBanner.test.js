import React from 'react';
import { shallow } from 'enzyme';
import IntroBanner from '../../../components/HomePage/IntroBanner';

describe('IntroBanner', () => {
  it('should render IntroBanner correctly', () => {
    const tree = shallow(<IntroBanner />);
    expect(tree).toMatchSnapshot();
  });
});
