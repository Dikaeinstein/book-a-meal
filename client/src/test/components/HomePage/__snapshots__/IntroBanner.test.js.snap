// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroBanner should render IntroBanner correctly 1`] = `
<section
  className="text-center intro-banner image-overlay text-white"
>
  <h1>
    Book-A-Meal
  </h1>
  <p
    style={
      Object {
        "margin": "1rem",
      }
    }
  >
    Your favourite meal order service
  </p>
</section>
`;
